lib/active_admin/application.rb

```diff
@@ -164,8 +164,7 @@ def controllers_for_filters
     private
 
     def register_default_assets
-      register_stylesheet "active_admin.css", media: "screen"
-      register_stylesheet "active_admin/print.css", media: "print"
+      register_stylesheet "active_admin.css"
```

Comment:
Here's the mistake I mentioned at https://github.com/activeadmin/activeadmin/pull/6922#discussion_r628010825.

This should be:

```suggestion
      register_stylesheet "active_admin.css", media: "all"
```

Reply:
Fix pushed!